hypothesis/via
# Authenticate with JSTOR using the site code passed by LMS

For:

 * https://github.com/hypothesis/via/issues/734
 
Requires:

 * https://github.com/hypothesis/lms/pull/3912

## Testing notes

It would be ideal to test this with a document which isn't publicly accessible, but I need to find one first.

This assumes you have the setup running from https://github.com/hypothesis/lms/pull/3912

 * Visit your JSTOR assigment with the F12 dev tools open
 * Inspect the data being pass to `proxied.pdf` - you should see `site_code=` in the query string
 * It should not be rejected by JSTOR

I've tried this with a restricted document (https://www.jstor.org/stable/j.ctt7zwbts) but we're getting upstream 500's?
